extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/SimpleJsonResource.java

```diff
@@ -47,7 +46,6 @@ public Person getPerson() {
         return person;
     }
 
-    @CustomSerialization(UnquotedFieldsPersonBiFunction.class)
```

Comment:
Why is this moved out? I don't see any test replacing this and it's confusing if the test still passes, no?

Reply:
It was not intended.

Reply:
fixed